element_calcium_imaging/imaging.py

```diff
@@ -848,7 +835,7 @@ def make(self, key):
             caiman_dataset = imaging_dataset
 
             # infer "segmentation_channel" - from params if available, else from caiman loader
-            params = (ProcessingParamSet * ProcessingTask & key).fetch1("params")
+            params = (ProcessingParamSet & key).fetch1("params")
```

Comment:
I think you do need the join with `ProcessingTask` here

Reply:
To cement my point, the primary keys of Curation are the following:

subject | session_id | scan_id | paramset_idx

Reply:
@ttngu207 You were right! ProcessingTask was required. I tested on the existing databases.